dwh/query_lib_big_query.py

```diff
@@ -0,0 +1,325 @@
+"""Module that implements Query Engine for BigQuery."""
+import typing as tp
+
+import pandas as pd
+
+from base import EncounterConstraints, ObsConstraints, PatientQuery
+
+
+_DATE_VALUE_SEPARATOR = ","
+
+try:
+    from google.cloud import bigquery
+except ImportError:
+    pass  # not all set up need to have bigquery libraries installed
+
+
+def _build_in_list_with_quotes(values: tp.Iterable[tp.Any]):
+    """
+    Builds the `in` operand for a where clause
+    """
+    return ",".join(('"{}"'.format(x) for x in values))
+
+
+class _BigQueryPatientQuery(PatientQuery):
```

Comment:
```suggestion
class BigQueryPatientQuery(PatientQuery):
```
given that this is used outside this module as well.